PrimerSDK.podspec

```diff
@@ -13,13 +14,13 @@ Pod::Spec.new do |spec|
     spec.swift_version = "4.2"
     spec.ios.deployment_target = '10.0'
     
-    spec.source_files = 'PrimerSDK/Classes/**/*'
+    spec.source_files = 'Sources/PrimerSDK/Classes/**/*'
     spec.resource_bundles = {
-        'PrimerSDK' => ['PrimerSDK/Assets/*.xcassets']
+        'PrimerSDK' => ['Sources/PrimerSDK/Resources/*.{strings,xcassets}']
```

Comment:
let's see how this goes when we deploy later. I've experienced a bunch of issues in the past when messing with these files and deploying to trunk :)